### DISCUSSION TITLE
Usar packages para acessibilidade

### DISCUSSION THREAD

**Discussion by GuilhermeSanchesS** [diego3g/rsxp-2023#24] [Ideas]
Como VLibras para tradução de textos do Português para Libras 
https://www.npmjs.com/package/@djpfs/react-vlibras

**Comment by Rodrigo001-dev** [comment]
Cara, adorei a ideai, podemos pensar em implementar

**Comment by diego3g** [comment]
Fala @GuilhermeSanchesS, não entendi muito bem como essa lib se encaixaria em nosso projeto. Pelo que entendi é uma lib web e não mobile, estou certo?

**Reply by GuilhermeSanchesS** [reply]
Sim, ela para web a ideia era usar como exemplo de package para acessibilidade não necessariamente VLibras alguma lib semelhante para react-native.
